tests/parser/parser.cpp

```diff
@@ -2040,3 +2076,7 @@ TEST(Parser, repeatedMathParsePrintBehaviourWithReset)
 
     EXPECT_EQ(in, output2);
 }
+
+TEST(Parser, parseEmptyEncapsulationWithId)
+{
+}
```

Comment:
What is this?...

Reply:
Finally, a test that passes!  ;)

Reply:
And will always pass no matter what. Best test ever!

Reply:
2635f862